src/loaders/data-sources/AnnDataSource.js

```diff
@@ -50,49 +50,49 @@ export default class AnnDataSource extends ZarrDataSource {
   constructor(...args) {
     super(...args);
     /** @type {Map<string, Promise<string[]>} */
-    this.cellSets = new Map();
+    this.obsPromises = new Map();
   }
 
   /**
-   * Class method for loading cell set ids.
+   * Class method for loading obs variables.
    * Takes the location as an argument because this is shared across objects,
    * which have different ways of specifying location.
-   * @param {string[]} cellSetZarrLocation An array of strings like obs.leiden or obs.bulk_labels.
+   * @param {string[]} obsLocations An array of strings like obs/leiden or obs/bulk_labels.
```

Comment:
Perhaps `obsPaths`? these are ultimately paths in the zarr store.

Reply:
Yes, better for sure.